### DISCUSSION TITLE
Using if/then statements to create multi-turn chatbot

### DISCUSSION THREAD

**Discussion by jfirma1** [FlowiseAI/Flowise#2061] [General]
I'm creating a flow where I am using prompt chaining to iterate on a prompt. It begins by the user asking a question, then the first chain breaks down the question into different parts, then the second chain comes up with the second part of the prompt and the third chain comes up with the third part of the prompt.

After this, the answer is displayed for the user.

However, if the user types out something in the chat window, like another question, I want this new question to go through an entirely new chain.

At the end of my first set of chains, I realize I could take the output prediction and use that as an input variable, however, I want the output prediction to be stored as a variable to be used later NOT as the trigger for the "if/then" logic.

My question is: How can I use multiple chains that get triggered by a user's question and not an output prediction?

**Comment by HenryHengZJ** [comment]
There is a Set Variable and Get Variable node. You can save the output prediction of LLMChain to Set Variable, then later retrieve it using Get Variable node

**Reply by jfirma1** [reply]
Thanks!  If I set a variable in one flow, can I retrieve it in another?

**Reply by HenryHengZJ** [reply]
No you can't, only within the same chatflow
